viewer/src/main/java/nl/b3p/viewer/stripes/EditFeatureActionBean.java

```diff
@@ -77,11 +78,11 @@
     @Validate
     private ApplicationLayer appLayer;
 
-    private Layer layer;
+    protected Layer layer;
 
-    private SimpleFeatureStore store;
+    protected SimpleFeatureStore store;
 
-    private JSONObject jsonFeature;
+    protected JSONObject jsonFeature;
```

Comment:
I would prefer using public getter/setter in an overriding class instead of making these `protected`

Reply:
I've got a class that inherits the EditFeatureActionBean, so this construction makes it somewhat more transparent than using a getter/setter